tests/async/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -18,7 +18,7 @@ class X {
 }
 
 =====================================output=====================================
-async function* a() {
+async function *a() {
   yield* b();
```

Comment:
@thorn0 Are you planning on doing the space between `yield *` in a separate PR or can we consider it related?

Reply:
I think we can.